Mathlib/Analysis/Normed/Affine/Isometry.lean

```diff
@@ -408,6 +429,38 @@ protected theorem continuousOn {s} : ContinuousOn e s :=
 protected theorem continuousWithinAt {s x} : ContinuousWithinAt e s x :=
   e.continuous.continuousWithinAt
 
+/-- Interpret a `LinearIsometryEquiv` as a `ContinuousLinearEquiv`. -/
```

Comment:
This comment, and one below, needs updating to reflect that this is an affine not a linear operation. And again I think this would better be in a separate PR.